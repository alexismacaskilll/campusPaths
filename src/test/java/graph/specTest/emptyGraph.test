# Tests that a graph is empty when it is created

# Create a graph
CreateGraph emptyGraph

# Print the nodes in the graph
ListNodes emptyGraph