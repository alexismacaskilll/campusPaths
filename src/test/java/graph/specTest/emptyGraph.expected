# Tests that a graph is empty when it is created

# Create a graph
created graph emptyGraph

# Print the nodes in the graph
emptyGraph contains: